pipelines/install/03-install_server.yaml

```diff
@@ -11,3 +11,5 @@
       scenario: "{{ pipeline_type }}"
       scenario_version: "{{ pipeline_version }}"
     - foreman_installer
+  post_tasks:
+    - shell: foreman-installer -v --katello-pulp-expose-db true
```

Comment:
This looks like another change that's not yet ready

Reply:
Doh, thought I cleaned my branch.